.github/workflows/Nightly.yml

```diff
@@ -16,7 +16,7 @@ permissions:
 
 jobs:
   tag-latest:
-    if: github.event_name == 'schedule'
+    if: ( github.repository == 'SimpleITK/SimpleITK.git' ) &&  ( github.event_name == 'schedule' )
```

Comment:
@hjmjohnson This should have been SimpleITK/SimpleITK without the .git. The builds last night were skipped.